nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -290,13 +290,71 @@ in
         type = types.bool;
         default = false;
         description = ''
-          Specifies whether sshd(8) should look up the remote host name, and to check that the resolved host name for
+          Specifies whether sshd should look up the remote host name, and to check that the resolved host name for
           the remote IP address maps back to the very same IP address.
           If this option is set to no (the default) then only addresses and not host names may be used in
           ~/.ssh/authorized_keys from and sshd_config Match Host directives.
         '';
       };
 
+      useTCPKeepAlive = mkOption {
```

Comment:
I think this is problematic from a maintenance point of view. I'd expect this to be auto-generated from some sources, which you don't seem to provide, which means that in two years it will be out of date and broken for someone.

Reply:
Me use official documentation - https://man.openbsd.org/sshd_config#TCPKeepAlive Renamed to tcpKeepAlive

Reply:
The concern is that you did a *manual* copy/paste of documentation, which means that if the source ever updates their data, that our module is out of date (and won't be updated unless someone does manual work). 

Renaming it doesn't address my raised concern. 

You might be lucky and it will still be merged regardless, but I'd count it as an increase in technical debt for the project.